GoogleChromeLabs/ps-analysis-tool
# Feature: Redesign CLI console messages and user interactions.

## Description
This PR aims to update the CLI messaging to provide an overall consistency to the output.

## Relevant Technical Choices
- Use individual validators for each option.
- Update messaging for CLI and update flags being used.
- Use `Coffee` library for providing an E2E framework for CLI.

## Testing Instructions
- Clone this branch.
- In the terminal run `npm run build:all`.
- Now run the CLI and check redesigned output messages.

## Additional Information:



## Screenshot/Screencast



---

## Checklist






Fixes #625
